Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -170,10 +170,15 @@ private void OnPurchaseMessage(EntityUid uid, ShipyardConsoleComponent component
             _accessSystem.TrySetTags(targetId, newAccess, newCap);
         }
 
-        var newDeed = EnsureComp<ShuttleDeedComponent>(targetId);
-        newDeed.ShuttleUid = shuttle.Owner;
-        newDeed.ShuttleName = name;
-        newDeed.ShuttleOwner = player;
+        var deedID = EnsureComp<ShuttleDeedComponent>(targetId);
+        deedID.ShuttleUid = shuttle.Owner;
+        deedID.ShuttleName = name;
+        deedID.ShuttleOwner = player;
+
+        var deedShuttle = EnsureComp<ShuttleDeedComponent>(shuttle.Owner);
+        deedShuttle.ShuttleUid = shuttle.Owner;
+        deedShuttle.ShuttleName = name;
+        deedShuttle.ShuttleOwner = player;
```

Comment:
why is this being doubled up?

Reply:
Not sure how will be the right way to fix it

Reply:
Used ChatGPT, I think its fine now.